cookbooks/software/redis/templates/default/redis-server.service.erb

```diff
@@ -0,0 +1,39 @@
+[Unit]
+Description=Advanced key-value store
+After=network.target
+Documentation=http://redis.io/documentation, man:redis-server(1)
+
+[Service]
+Type=forking
+ExecStart=/usr/bin/redis-server /etc/redis/redis.conf
+PIDFile=/var/run/redis/redis-server.pid
+TimeoutStopSec=0
+Restart=always
+User=redis
+Group=redis
+
+ExecStartPre=-/bin/run-parts --verbose /etc/redis/redis-server.pre-up.d
```

Comment:
[This service file](https://gist.github.com/mkocikowski/aeca878d58d313e902bb) looks a little bit simpler to me. I really don't think these pre-hooks are needed (or even used). They're not even [in this cheatsheet](https://www.digitalocean.com/community/tutorials/understanding-systemd-units-and-unit-files). 

But I might be proven wrong, if it works then that's OK with me :)